### PR TITLE
feat : added button and badge pages

### PR DESCRIPTION
**Pull request by Kevin-Solomon**
Just made an button and badge page.
Please let me know your feedback.

### CODE REVIEW

File: pages/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+@import url(../avatar/avatar.css);
+.badge {
```
Comment (VishalPatil18) [inline pages/badge/badge.css:2]: **Fix:** You can change the color of the text inside the badge to something light, as black on red is creating contrast issues.
Screenshot 2022-01-28 at 20 42 30

**Review by VishalPatil18 — CHANGES_REQUESTED** [review]
The buttons look quite cool, I liked them! Apart from one fix in the badge, rest looks quite good 👍

File: pages/badge/badge.css (hunk 1 above)
Comment (Kevin-Solomon) [reply pages/badge/badge.css:2]: thanks you for the feedback will impliment this change right away

**Review by VishalPatil18 — APPROVED** [review]
Great! Now the badge too looks cool!

**Review by ChhikaraBRUH — APPROVED** [review]
Code looks great

**Pull request merged**